app/components/track-page/resume-track-button.ts

```diff
@@ -27,16 +27,17 @@ export default class CourseOverviewResumeTrackButtonComponent extends Component<
     }
 
     return this.args.courses.find((course) => {
-      return !this.authenticator
-        .currentUser!.repositories.filterBy('course', course)
-        .filterBy('language', this.args.language)
-        .some((repository) => repository.allStagesAreComplete);
+      const courseRepositories = this.authenticator.currentUser!.repositories.filterBy('course', course).filterBy('language', this.args.language);
+
+      return courseRepositories.length > 0 && !courseRepositories.some((repository) => repository.allStagesAreComplete);
```

Comment:
There's a subtle bug for course with zero repos:
- `![].some` is true, 
- but we don't want to resume a course with zero repos.

Reply:
@andy1li Can you clarify this? `[].some()` shouldn't return true if there are no elements (as far as I know).

Reply:
Oh... k got it, `some` returns false, but there are zero repos. Cool!